### ISSUE TITLE
what sorts of loss values do you see?

### ISSUE THREAD

**Issue by el3ment** [chaiyujin/glow-pytorch#5]
In re-implementing this, I am finding it difficult to drop lower than 9ish - this seems like it's inevitable since the loss term is made up of (ignoring the pixel factor)
```
loss = -(loglikelihood_prior + logdet + discretization_correction) / log(2)
```
if z == 0, then the loglikelihood_prior is maximized, and you get a constant "-0.5 * log(2pi)"
thanks to the `sigmoid(scale + 2)` the logdet must always be negative, or 0 in the "best case", 
and the discretization correction is a constant negative

I don't see how it's possible to get a loss of "3" or "1" seen in the glow paper graphs

**Comment by chaiyujin** [comment]
![loss](https://user-images.githubusercontent.com/10378898/46990132-736ea080-d132-11e8-8b29-b59ffb0d2ab0.png)

**Comment by el3ment** [comment]
I was not implementing ActNorm, which produces positive logdet terms.
